CMakeLists.txt

```diff
@@ -0,0 +1,179 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(opus LANGUAGES C)
```

Comment:
You can remove `LANGUAGES`. `C` and `CXX` are enabled by default.

Reply:
If I use "LANGUAGES C" it will only check for c compiler and the generation will be faster, so will leave this one